index.html

```diff
@@ -2315,9 +2335,10 @@ <h2 id="docconformance">
               <p>
                 Role: <a href="#index-aria-menu">`menu`</a>
               </p>
+              <p>Authors SHOULD NOT use the `aria-multiselect` attribute on a `select` element.</p>
```

Comment:
Incorrect: aria-multiselect
Correct: aria-multiselectable

Reply:
thank you @mtrootyy 

silly mistake on my part, was trying to get this wrapped up too quickly